src/LogViewer/chunk-worker.js

```diff
@@ -128,14 +128,21 @@ const update = (response) => {
 const error = () => self.postMessage(JSON.stringify({ type: 'error' }));
 const loadEnd = () => self.postMessage(JSON.stringify({ type: 'loadend' }));
 
-const toHtml = (index) => {
+const getParagraphClass = (lineNumber, metadata) => {
+  const { highlightStart, highlightEnd } = metadata;
+
+  return lineNumber >= highlightStart && lineNumber <= highlightEnd ? 'highlight' : '';
+};
+
+const toHtml = (index, metadata) => {
   const lines = decode(chunks[index]);
   const start = LINE_CHUNK * index + offset + 1;
 
   const html = lines.map((parts, index) => {
     const lineNumber = start + index;
+    const pClass = getParagraphClass(lineNumber, metadata);
 
-    return `<p><a id="${lineNumber}">${lineNumber}</a>${parts.map((part) => {
+    return `<p class="${pClass}"><a id="${lineNumber}">${lineNumber}</a>${parts.map((part) => {
```

Comment:
I added the space in the `getParagraphClass` function so it would be rendered optionally.